src/java.desktop/windows/classes/sun/print/Win32PrintJob.java

```diff
@@ -437,18 +432,7 @@ public void print(Doc doc, PrintRequestAttributeSet attributes)
 
             if (mDestination != null) { // if destination attribute is set
                 try {
-                    FileOutputStream fos = new FileOutputStream(mDestination);
-                    byte []buffer = new byte[1024];
-                    int cread;
-
-                    while ((cread = instream.read(buffer, 0, buffer.length)) >=0) {
-                        fos.write(buffer, 0, cread);
-                    }
-                    fos.flush();
-                    fos.close();
-                } catch (FileNotFoundException fnfe) {
-                    notifyEvent(PrintJobEvent.JOB_FAILED);
-                    throw new PrintException(fnfe.toString());
+                    Files.copy(instream, Path.of(mDestination));
```

Comment:
Looks like the new code unlike the old one will throw an exception if the file is exists already. WIll it affect the printing functionality?

Reply:
good catch! Added `StandardCopyOption.REPLACE_EXISTING` to preserve behavior.